roles/kubevirt/tasks/provision.yml

```diff
@@ -65,25 +65,25 @@
       stat:
         path: "{{ offline_template_dir }}/v{{ version }}/demo-content.yaml"
       register: offline_demo_content
-      when: byo_demo_content.stat.exists == False
+      when: byo_demo_content.stat.exists == false
 
     - name: Download Demo Content
       get_url:
         url: "{{ release_manifest_url }}/v{{ version }}/demo-content.yaml"
         dest: "{{ kubevirt_template_dir }}/demo-content.yaml"
-      when: byo_demo_content.stat.exists == False and offline_demo_content.stat.exists == False
+      when: byo_demo_content.stat.exists == false and offline_demo_content.stat.exists == false
 
     - name: Copy Offline Demo Content to /tmp
       copy:
         src: "{{ offline_template_dir }}/v{{ version }}/demo-content.yaml"
         dest: "/tmp/demo-content.yaml"
-      when: offline_demo_content is not skipped and offline_demo_content.stat.exists == True
+      when: offline_demo_content is not skipped and offline_demo_content.stat.exists == true
 
     - name: Copy BYO Demo Content to /tmp
       copy:
         src: "{{ kubevirt_template_dir }}/demo-content.yaml"
         dest: "/tmp/demo-content.yaml"
-      when: byo_demo_content.stat.exists == False and (offline_demo_content is skipped or offline_demo_content.stat.exists == False)
+      when: byo_demo_content.stat.exists == false and (offline_demo_content is skipped or offline_demo_content.stat.exists == false)
```

Comment:
are you sure that it is string ? I can read in documentation that it is boolean https://docs.ansible.com/ansible/2.7/modules/stat_module.html#return-values

?